caffe2/CMakeLists.txt

```diff
@@ -229,6 +229,10 @@ else()
 target_compile_options(caffe2 INTERFACE "$<$<COMPILE_LANGUAGE:CXX>:-std=c++11>")
 endif()
 
+# Note(jiayq): This is not complete yet, but in the end we will need to deal with
+# explicit hidden visibility.
+target_compile_options(caffe2 PRIVATE "-fvisibility=hidden")
```

Comment:
Note: this line is here just for testing. While we should eventually turn it on, this diff is not ready to turn it fully on yet.